regex-assembly/include/php-rce-bypass.ra

```diff
@@ -0,0 +1,30 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##!> define single_or_double_quotes ['"]
+##!> define string_in_parens \(.+\)
```

Comment:
Add comment that you want to match potential intermediate parentheses with the any match, e.g. `(("sys")."t".("em"))("ls");`.

Reply:
done